src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/EndpointResource.cs

```diff
@@ -110,5 +111,17 @@ private Uri BuildUri(ResourceRequestContext context)
             builder.Scheme = Protocol;
             return builder.Uri;
         }
+
+        private static string Getfqdn()
+        {
```

Comment:
We'll need to pass BridgeConfiguration in if this becomes a property. So we may not want to cache it as a static.